src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysis.cs

```diff
@@ -2,6 +2,7 @@
 
 using System;
 using System.Collections.Immutable;
+using System.Threading;
```

Comment:
Nit: Why add this using?

Reply:
Hmm looks like I probably added something and reverted, but forgot to revert the using. Will remove, thanks.